pkgs/misc/emulators/mednaffe/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, pkgconfig, gtk2, mednafen }:
+
+stdenv.mkDerivation rec {
+
+  name = "mednaffe-${meta.version}";
+
+  src = fetchurl {
+    url = "https://github.com/AmatCoder/mednaffe/archive/v${meta.version}.tar.gz";
```

Comment:
Please use `fetchFromGitHub` here instead of `fetchurl` because the tarball checksums tend to change as Github regenerates them. The former hashes the contents not only the tarball.